frontend/pages/policies/PolicyPage/components/PolicyForm/PolicyForm.tsx

```diff
@@ -294,7 +294,7 @@ const PolicyForm = ({
     }
 
     return (
-      <Button variant="small-icon" onClick={onOpenSchemaSidebar}>
```

Comment:
The height of `"small-icon"` was making the label/sql box jump in spacing for the policy form. This was already properly `"text-icon"` with no bugginess for the label and query pages.